first attempt at refactoring README (#10742)
spectrocloud/librarium: docs/contributing/generated-content.md

```shell
export DISABLE_SECURITY_INTEGRATIONS=true
```

### Cached Security Bulletins

All CVE related data is saved to a
[GitHub Workflow Artifact](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow#about-workflow-artifacts)
after every successful release to production. Check out the
[post_release.yaml](../../.github/workflows/post_release.yaml) for further details. The cached data enables us to build
and start librarium without performing any CVE-related API queries. All of our GitHub workflows will use this cached
data as a fallback in the case of an API related build failure. Check out the
[build-cached-cves action.yaml](../../.github/actions/build-cached-cves/action.yaml) to learn how the cached data is
fetched and used.

CVE data is saved locally in the `.docusaurus/security-bulletins/default/data.json` file. You can remove the data using
`make clean-security`. You can use the cached CVE artifact locally when you want to avoid the CVE download time. This
flow also helps you when you don't have any local CVE data and we are experiencing an API outage.

> librarium provides the following commands which fetch cached CVE data to your local environment.

Reviewer: ```suggestion
Librarium provides the following commands that fetch cached CVE data into your local environment.
```

### Cached Packs Data

All pack related data is saved to a
[GitHub Workflow Artifact](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow#about-workflow-artifacts)
after every successful release to production. Check out the
[post_release.yaml](../../.github/workflows/post_release.yaml) for further details. The cached data enables us to build
and start librarium without performing any pack related API queries. All of our GitHub workflows will use this cached
data as a fallback in the case of an API related build failure. Check out the
[build-cached-packs action.yaml](../../.github/actions/build-cached-packs/action.yaml) to learn how the cached data is
fetched and used.

Packs data is saved locally in the `.docusaurus/packs-integrations` and `static/img/packs` folders. You can remove the
data using `make clean-packs`. You can use the cached packs artifact locally when you don't have any downloaded packs
data and you want to avoid the pack download time. This flow also helps you when you don't have any local packs data and
we are experiencing an API outage.

> librarium provides the following commands which fetch cached packs to your local environment.

Reviewer: ```suggestion
Librarium provides the following commands, which fetch cached packs to your local environment.
```
